dspace/bitstream.py

```diff
@@ -7,10 +7,11 @@
 from typing import Optional
 
 from requests import Response
+from smart_open import open
```

Comment:
Just to make it _extra_ clear that the `open()` function below is not using the standard library, I would `import smart_open` here and then call `smart_open.open()` below.